contracts/ethregistrar/StablePriceOracle.sol

```diff
@@ -4,95 +4,138 @@ import "./PriceOracle.sol";
 import "./SafeMath.sol";
 import "./StringUtils.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/introspection/IERC165.sol";
 
 interface AggregatorInterface {
-  function latestAnswer() external view returns (int256);
+    function latestAnswer() external view returns (int256);
 }
 
-
 // StablePriceOracle sets a price in USD, based on an oracle.
-contract StablePriceOracle is Ownable, PriceOracle {
+contract StablePriceOracle is PriceOracle {
     using SafeMath for *;
     using StringUtils for *;
 
-    // Rent in base price units by length. Element 0 is for 1-length names, and so on.
-    uint[] public rentPrices;
+    // Rent in base price units by length
+    uint256 public immutable price1Letter;
+    uint256 public immutable price2Letter;
+    uint256 public immutable price3Letter;
+    uint256 public immutable price4Letter;
+    uint256 public immutable price5Letter;
 
     // Oracle address
     AggregatorInterface public usdOracle;
```

Comment:
Can this also be immutable?

Reply:
Yes! I must have left that out whilst testing gas efficiency. Good catch.